frontend/rubenpong/package.json

```diff
@@ -17,14 +17,15 @@
     "@fortawesome/free-regular-svg-icons": "^6.3.0",
     "@fortawesome/free-solid-svg-icons": "^6.3.0",
     "@fortawesome/vue-fontawesome": "^3.0.3",
+    "axios": "^1.3.4",
```

Comment:
Why is axios used in the frontend? We use `fetch` to communicate with the backend.